README.md

```diff
@@ -1243,3 +1243,78 @@ out  | evt.ota_end.report      | object     | Sent on upgrade end with upgrade s
    }
 }
 ```
+
+### Zwave Specific service 
+This service is used to meet all zwave specific requirements that are not satisfied with already implemented services.
+
+#### Service name 
+
+`zwave_specific`
+
+#### Interfaces
+
+Type | Interface               | Value type | Description 
+-----|-------------------------|------------|--------------
+out  | evt.notification.report | str_map    | Notification Reports that are sent by slave devices.
+
+#### Storage property
+
+Key        | Value description 
+-----------|----------------------------------
+sub_value  | Values pair to store in Vinculum.
+
+#### Examples
+
+Motion Detection occurs for state Motion Sensor Status:
+
+```json
+{
+   "type": "evt.notification.report",
+   "serv": "zwave_specific",
+   "val_t": "str_map",
+   "val": {
+      "notificationEventState": "motion_detection",
+      "notificationType": "home_security",
+      "stateVariable": "motion_sensor_status"
```

Comment:
If these are zwave specific key names, can you please dcoument what they represent? It's not immediately clear to me what they are.

If they are not zwave specific and they are keys we have added, can we revise the names?

notificationType: type of the notification, gotcha.
notificationEventState: why not just notificationEvent?
stateVariable: why not just notificationState

Either way, these are not that clear and should be documented / revised.

Reply:
fixed, use proper description now